### PR TITLE
Standardize Phoenix Setup

### PR DESCRIPTION
**Pull request by cathyzoller**
- adds setup script
- adds changesets to schemas
- forked from [cmb/switch-migrations-to-phoenix](https://github.com/mojotech/helios2/tree/cmb/switch-migrations-to-phoenix)

### CODE REVIEW

File: server-phoenix/lib/helios/event.ex [hunk 1]
```diff
@@ -23,32 +31,32 @@ defmodule Helios.Event do
   end
 
   def created_after(query, c_after) when is_struct(c_after) do
-    from q in query, where: q.created_at > ^c_after
+    from(q in query, where: q.inserted_at > ^c_after)
```
[the remark this answers is not in the document]
Comment (cathyzoller) [reply server-phoenix/lib/helios/event.ex:34]: I forgot to change these back to :created_at... also, the visual studio code elixir extension I was using recently added the obligatory parens (ugh).  I changed extensions but missed a few of these.  Will fix.

File: server-phoenix/lib/helios/widget.ex [hunk 2]
```diff
@@ -37,21 +52,23 @@ defmodule Helios.Widget do
...
       join: s in subquery(current_position),
       where: q.position > s.position,
       order_by: q.position,
       limit: 1
+    )
   end
 
   def default(query) do
-    from q in query, order_by: q.position, limit: 1
+    from(q in query, order_by: q.position, limit: 1)
```
[the remark this answers is not in the document]
Comment (cathyzoller) [reply server-phoenix/lib/helios/widget.ex:72]: Yes, we can add a formatter.exs file with our preferences.  I prefer using parens with ecto.from.  Others do not.  I just checked source_advisors, and Jeff & Elias are also using the parens in this situation.

[the remark this answers is not in the document]
Comment (cathyzoller) [reply server-phoenix/lib/helios/widget.ex:72]: The mix formatter uses the parens in Ecto.Query.from by convention, which is why they were added here.  All of the files in /lib/**/*.ex are automatically formatted  on save with mix format in VScode, except when there's a glitch.

File: server-phoenix/lib/helios/event.ex (hunk 1 above)
[the remark this answers is not in the document]
Comment (cathyzoller) [reply server-phoenix/lib/helios/event.ex:34]: Resolved

**Comment by cathyzoller** [comment]
@cbarber I added a commit to update the erlang otp to use 24.1.7.  I can remove if it breaks the heroku setup (It shouldn't, since source_advisors is using the same version, but no guarantees.)

**Review by cbarber — APPROVED** [review]
Nice work :rocket:

**Pull request merged**